spec/services/service_provider_updater_spec.rb

```diff
@@ -10,7 +10,8 @@
     [
       {
         id: 'big number',
-        updated_at: '2010-01-01 00:00:00',
+        created_at: '2010-01-01 00:00:00'.to_datetime,
+        updated_at: '2010-01-01 00:00:00'.to_datetime,
```

Comment:
I was trying to make sure that if the attributes were not cleaned up, the test would fail, but it doesn't fail when the value is just a String because when the ServiceProvider gets saved to the DB, Rails converts it to a DateTime, but the test was trying to compare a DateTime with a String, so the comparison will always return false, hence making the test pass, which is a false positive.